helm/oauth2-proxy/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 6.2.7
+version: 6.2.8
```

```suggestion
version: 6.3.0
```
(Or whatever needed version after rebase): this is new feature according to semver.